apps/src/code-studio/pd/workshop_dashboard/components/survey_results_header.jsx

```diff
@@ -331,7 +331,9 @@ const SurveyResultsHeader = React.createClass({
   },
 
   getCoursesForWorkshops() {
-    return _.uniq(_.flatMap(this.props.workshops, workshop => workshop.course));
+    let courses = _.uniq(_.flatMap(this.props.workshops, workshop => workshop.course));
+    _.remove(courses, (course) => {return ['Admin', 'Counselor'].indexOf(course) >= 0;});
+    return courses;
```

Comment:
I don't really like doing Array.from( new Set()) to create an array of unique elements - doesn't seem as intuitive reading the code.

I like the second line